### PR TITLE
PMM-6389 Add VictoriaMetrics

### PR DESCRIPTION
**Pull request by valyala**
- [x] https://github.com/percona/pmm-server-packaging/pull/156
- [ ] https://github.com/Percona-Lab/percona-images/pull/82
- [ ] https://github.com/percona/pmm-update/pull/96
- [ ] https://github.com/percona/pmm-managed/pull/447

### CODE REVIEW

File: ansible/roles/pmm2-images/tasks/main.yml [hunk 1]
```diff
@@ -107,6 +107,7 @@
         name:
           - percona-grafana
           - percona-prometheus
+          - percona-VictoriaMetrics
```
Comment (EvgeniyPatlan) [inline ansible/roles/pmm2-images/tasks/main.yml:110]: in line 3 of this file:
`# All new plays should be added into pmm-update playbook. It is invoked after pmm2 playbook.`

Comment (valyala) [reply ansible/roles/pmm2-images/tasks/main.yml:110]: Got it! See https://github.com/percona/pmm-update/pull/96 .

Comment (valyala) [reply ansible/roles/pmm2-images/tasks/main.yml:110]: Abandoning this PR because of https://github.com/percona/pmm-update/pull/96

Comment (valyala) [reply ansible/roles/pmm2-images/tasks/main.yml:110]: Got it! See https://github.com/percona/pmm-update/pull/96

Comment (AlekSi) [reply ansible/roles/pmm2-images/tasks/main.yml:110]: After a lot of discussions and attempts to fix things, we decided that new packages should be added in both places – there and in pmm-update

Comment (AlekSi) [reply ansible/roles/pmm2-images/tasks/main.yml:110]: `percona-victoriametrics`, lowercase?

**Review by AlekSi — CHANGES_REQUESTED** [review]

File: ansible/roles/pmm2-images/tasks/main.yml (hunk 1 above)
Comment (valyala) [reply ansible/roles/pmm2-images/tasks/main.yml]: done

**Review by EvgeniyPatlan — APPROVED** [review]

**Review by Sudokamikaze — APPROVED** [review]

**Pull request merged**